src/phoenix/session/client.py

```diff
@@ -88,15 +88,12 @@ def __init__(
             )
         if kwargs:
             raise TypeError(f"Unexpected keyword arguments: {', '.join(kwargs)}")
-        headers = dict((headers or get_env_client_headers() or {}))
+        headers = dict(headers or {})
         if api_key:
             headers = {
                 **{k: v for k, v in (headers or {}).items() if k.lower() != "authorization"},
-                "Authorization": f"Bearer {api_key}",
+                "authorization": f"Bearer {api_key}",
```

Comment:
Need to leave this unchanged for hosted Phoenix

Reply:
good catch, thank you

Reply:
I think this isn't the hosted Phoenix header, we manage that separately, this capitalization doesn't matter afaik